internal/collector/sysinfo/platform/platform_linux.go

```diff
@@ -184,14 +195,41 @@ func (p Collector) getWSLVersion() uint8 {
 	return 1
 }
 
+// getKernelVersion returns the kernel version of the system.
+func (p Collector) getKernelVersion() string {
+	kVersion := fileutils.ReadFileLogError(filepath.Join(p.platform.root, "proc/version"), p.log)
```

Comment:
nitpick: We typically avoid meaningful names for short lived variables where the context makes them self-explanatory. `kVersion` here could just be `v`, and `strings` could be `f` (for fields) or `s` (for strings or split).
I don't have a strong opinion about this, but I've seen didrocks mentioning this quite a lot ;)

Reply:
Also, naming your variable as `strings` shadows the package, such that you would not be able to call other function from that package while that variable remains in scope.